### PR TITLE
docs: Added docs regarding remote execution plane (#151)

### LINKED ISSUE #151: Document setup for remote control plane
While in https://github.com/keptn-contrib/job-executor-service/issues/148 (PR: https://github.com/keptn-contrib/job-executor-service/pull/149) we have added installation details using `helm install` for the case of installing job-executor-service in Keptn control-plane, we are lacking details for installing it as a remote execution plane service.

This setup would look as follows:
```bash
helm install ... --set remoteControlPlane.enabled=true,remoteControlPlane.api.protocol=https,remoteControlPlane.api.hostname=<KEPTN-API-HOST>,remoteControlPlane.api.token=<KEPTN-API-TOKEN>
```

# Definition of Done

- [ ] Latest version of job-executor-service installed in a remote-execution plane setup and manually tested with Keptn 0.10.0
- [ ] Documentation added to README

### PR DESCRIPTION
**Pull request by christian-kreuzberger-dtx**
## This PR

- Adds docs for remote execution plane docs

Fixes #151

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,36 @@ Please replace `<VERSION>` with the actual version you want to install from the
...
+* `remoteControlPlane.topicSubscription` - list of Keptn CloudEvent types that this instance should listen to, e.g., `sh.keptn.event.remote-task.triggered`
+* `remoteControlPlane.api.protocol` - protocol (`http` or `https`) used to connect to the remote control plane
+* `remoteControlPlane.api.hostname` - Keptn API Hostname (e.g., `1.2.3.4.nip.io`)
+* `remoteControlPlane.api.token` - Keptn API Token (can be obtained from Bridge)
+
+**Example**
+```bash
+KEPTN_API_PROTOCOL=http # or https
+KEPTN_API_HOST=<INSERT-YOUR-HOSTNAME-HERE> # e.g., 1.2.3.4.nip.io
+ KEPTN_API_TOKEN=<INSERT-YOUR-KEPTN-API-TOKEN-HERE>
```
Comment (Paolo-Chila) [inline README.md:53]:
```suggestion
KEPTN_API_TOKEN=<INSERT-YOUR-KEPTN-API-TOKEN-HERE>
```

File: README.md [hunk 2]
```diff
@@ -33,6 +33,36 @@ Please replace `<VERSION>` with the actual version you want to install from the
...
+
+TASK_SUBSCRIPTION=sh.keptn.event.remote-task.triggered
+
+helm install -n <NAMESPACE> \
+  job-executor-service https://github.com/keptn-contrib/job-executor-service/releases/download/<VERSION>/job-executor-service-<VERSION>.tgz \
```
Comment (Paolo-Chila) [inline README.md:58]:
```suggestion
 job-executor-service https://github.com/keptn-contrib/job-executor-service/releases/download/<VERSION>/job-executor-service-<VERSION>.tgz \
```

**Review by Paolo-Chila — APPROVED** [review]
A couple of extra whitespaces, everything else looks good otherwise

File: README.md (hunk 1 above)
Comment (christian-kreuzberger-dtx) [reply README.md:53]: this is on purpose. if you prefix it with a ` ` (space), it won't turn up in the (bash) history.

File: README.md (hunk 2 above)
Comment (christian-kreuzberger-dtx) [reply README.md:58]: The space here is also on purpose for formatting reasons (there is a `\` and a newline)

**Pull request merged**